lib/ansible/modules/storage/netapp/na_ontap_object_store.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python
+
+# (c) 2019, NetApp, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+
+DOCUMENTATION = '''
+
+module: na_ontap_object_store
+short_description: NetApp ONTAP manage object store config.
+extends_documentation_fragment:
+    - netapp.na_ontap
+version_added: '2.9'
+author: NetApp Ansible Team (@carchi8py) <ng-ansibleteam@netapp.com>
+
+description:
+- Create or delete object store config on ONTAP.
+
+options:
+
+  state:
+    description:
+    - Whether the specified object store config should exist or not.
+    choices: ['present', 'absent']
+    default: 'present'
+
```

Comment:
https://github.com/ansible/ansible/pull/59048#issuecomment-511038554

Reply:
CI was complaining about it above. I think it is in the arg spec but wasn't in docs. Now that I check the arg spec, it isn't there and isn't in any other module.

Reply:
found it @lonico since it isn't on the ignore list (some might be), since it uses a `default` its need to have type string. [link](https://docs.ansible.com/ansible/devel/dev_guide/testing_validate-modules.html#errors) error #338